src/Appwrite/Platform/Workers/Webhooks.php

```diff
@@ -113,7 +120,19 @@ private function execute(array $events, string $payload, Document $webhook, Docu
         }
 
         if (false === \curl_exec($ch)) {
-            $this->errors[] = \curl_error($ch) . ' in events ' . implode(', ', $events) . ' for webhook ' . $webhook->getAttribute('name');
+            $errorCount = $webhook->getAttribute('errors', 0) + 1;
+            $lastErrorLogs = \curl_error($ch) . ' in events ' . implode(', ', $events) . ' for webhook ' . $webhook->getAttribute('name');
+            $webhook->setAttribute('errors', $errorCount);
+            $webhook->setAttribute('logs', $lastErrorLogs);
+
+            if ($errorCount > 9) {
```

Comment:
9 here should be const, or env variable. Check with Christy which one he prefers.